src/encoded/schemas/atac_frip_quality_metric.json

```diff
@@ -0,0 +1,79 @@
+{
+    "title": "ATAC-seq FRiP quality metric",
+    "description": "Schema for reporting ATAC-seq fraction of reads in peaks quality metrics",
+    "id": "/profiles/atac_frip_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run","quality_metric_of", "award", "lab"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "FRiP": {
+            "title": "FRiP score of peaks",
+            "description": "Fraction of reads in the peak file",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "pr1_FRiP": {
+            "title": "FRiP score of pseudoreplicate 1 peak",
+            "description": "FRiP score of pseudoreplicate 1 peak of the corresponding replicate of this peak.",
```

Comment:
title: FRiP score of pseudoreplicate 1
description: Fraction of reads in the peak file of pseudoreplicate 1